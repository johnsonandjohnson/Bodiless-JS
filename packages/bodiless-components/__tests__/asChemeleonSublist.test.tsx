import React from 'react';

import { render } from 'enzyme';
import TestList from './TestChameleonList';

describe('asChameleonSubList', () => {
  it('works', () => {
    const $ = render(<TestList />);
    expect($).toMatchSnapshot();
  });
});
