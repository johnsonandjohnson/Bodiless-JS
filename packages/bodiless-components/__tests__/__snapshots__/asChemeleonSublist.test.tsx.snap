// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`asChameleonSubList works 1`] = `
<ul>
  <li
    data-bl-activator="true"
  >
    <span>
      Top
    </span>
    <ul>
      <li
        class="list-disc ml-5"
        data-bl-activator="true"
      >
        <span>
          Bulleted
        </span>
        <ul>
          <li
            class="list-decimal ml-5"
          >
            <span>
              Numbered
            </span>
          </li>
        </ul>
      </li>
    </ul>
  </li>
</ul>
`;
